### DISCUSSION TITLE
Jak usunąć/wyłączyć/ukryć sekcję komentarzy pod kontaktem?

### DISCUSSION THREAD

**Discussion by rbtm** [YetiForceCompany/YetiForceCRM#15836] [How to]
Dzień dobry, nie wiem w którym miejscu szukać i czy w ogóle jest możliwość wyłączenia zakładki, którą zaznaczyłem na zrzucie? To Komentarze pod osobą kontaktową. Nie mogę wyłączyć modułu jako takiego, bo działa również dla kontrahenta, pola to nie ten poziom, a mi chodzi o to, żeby pracownik nie dodawał mi oddzielnych komentarzy pod osobami kontaktowymi, gdyż wszystko robimy na głównej karcie kontahenta. Dodawanie komentarzy na tym poziomie powoduje tylko ich "ukrywanie". Proszę o wskazówkę, w którym miejscu szukać rozwiązania, co zrobić, żeby ta zakładka zniknęła? Dzięki

![Screenshot 2022-03-31 at 08-54-32 Kontakty Osoba Testowa](https://user-images.githubusercontent.com/58072546/160995886-a66877c9-3c51-485c-8fc7-75309dff3aaf.png)

**Comment by bpabiszczak** [comment]
tylko programistycznie.
